src/vs/workbench/contrib/snippets/test/browser/snippetsService.test.ts

```diff
@@ -68,6 +71,11 @@ suite('SnippetsService', function () {
 		)]);
 	});
 
+	teardown(function () {
+		for (const disposable of disposables) {
+			disposable.dispose();
+		}
```

Comment:
would then be `store.clear()`

Reply:
I refactored this a bit so there's only one DisposableStore for the whole test suite and then I dispose it in suiteTeardown